interop/test_utils.go

```diff
@@ -387,10 +387,10 @@ func DoPerRPCCreds(tc testgrpc.TestServiceClient, serviceAccountKeyFile, oauthSc
 		FillUsername:   true,
 		FillOauthScope: true,
 	}
-	token := GetToken(serviceAccountKeyFile, oauthScope)
+	token := GetToken(ctx, serviceAccountKeyFile, oauthScope)
 	kv := map[string]string{"authorization": token.Type() + " " + token.AccessToken}
-	ctx := metadata.NewOutgoingContext(context.Background(), metadata.MD{"authorization": []string{kv["authorization"]}})
-	reply, err := tc.UnaryCall(ctx, req)
+	outCtx := metadata.NewOutgoingContext(ctx, metadata.MD{"authorization": []string{kv["authorization"]}})
+	reply, err := tc.UnaryCall(outCtx, req)
```

Comment:
This can just shadow or replace `ctx` instead of making a new thing called `outCtx`.